src/components/codemirror-utils.js

```diff
@@ -0,0 +1,25 @@
+export function getAllSelections(doc) {
```

Comment:
This feels like a function that should have at least a few tests.

Reply:
i agree in principle, but i'm not sure it's best to invest in cleaning up pre-existing technical debt around code mirror in this PR (this is just one example of the zillion function in our codebase that has no tests -- in general i've tried to leave things better than i found them by adding tests around some action creators that didn't have any previously, and that's what i've been shooting for in my last few PRs -- adding a few new tests for old code, and testing  my new code)

Reply:
Well, I can't really force you to do anything, but I would encourage you to write at least one or two tests for this function in particular. Refactoring tends to be exactly when you break things unexpectedly, and writing tests is some insurance against that. Especially in a case like this where I suspect writing a test should only take 10 minutes or so.